Update capture-image-portal.md (#119924)
Azure/azure-content: articles/virtual-machines/capture-image-portal.md

1. Go to the [Azure portal](https://portal.azure.com), then search for and select **Virtual machines**.

2. Select your VM from the list.

>    If you want to proceed with a generalized image, please follow steps to generalize the VM [Generalize OS disk for Linux/Windows](https://learn.microsoft.com/azure/virtual-machines/generalize)

Reviewer: ```suggestion
- If you want a generalized image, see [Generalize OS disk for Linux/Windows](https://learn.microsoft.com/azure/virtual-machines/generalize).

```

>    If you want a specialized image, no additional action required,Kindly proceed to Step 3.

Reviewer: ```suggestion
- If you want a specialized image, no additional action is required.

```

1. Go to the [Azure portal](https://portal.azure.com), then search for and select **Virtual machines**.

2. Select your VM from the list.

> - If you want a generalized image, see [Generalize OS disk for Linux/Windows](https://learn.microsoft.com/azure/virtual-machines/generalize).

Reviewer 2: ```suggestion
- If you want a generalized image, see [Generalize OS disk for Linux/Windows](/azure/virtual-machines/generalize).

```

>    If you want a specialized image, no additional action required,Kindly proceed to Step 3.

Reviewer: ```suggestion
- If you want a specialized image, no additional action is required.
```

Reviewer 3: In addition to Julia's review feedback, please fix the reference link with an absolute path.

1. Go to the [Azure portal](https://portal.azure.com), then search for and select **Virtual machines**.

2. Select your VM from the list.

> - If you want a generalized image, see [Generalize OS disk for Linux/Windows](/azure/virtual-machines/generalize).

Reviewer 2: ```suggestion
   - If you want a generalized image, see [Generalize OS disk for Linux/Windows](/azure/virtual-machines/generalize).

```

> - If you want a specialized image, no additional action is required.

Reviewer 2: ```suggestion
   - If you want a specialized image, no additional action is required.

```
